packages/block-editor/src/components/block-list/block.js

```diff
@@ -121,6 +121,7 @@ function BlockListBlock( {
 			'is-focus-mode': isFocusMode,
 			'has-child-selected': isAncestorOfSelectedBlock,
 			'is-block-collapsed': isAligned,
+			'is-focus-container': name === 'core/template-part',
```

Comment:
We need to create this class in a more extensible way. Before, with `InnerBlocks`, we added our custom styles if InnerBlocks had the `onInput` prop and if it or one of its children was selected.